### PR TITLE
Replaces apiService with axios

### PR DESCRIPTION
**Pull request by mcousillas6**
Same PR as on the react-native base.

Note: added the `parseError` utility to the project as well, since we weren't parsing backend errors properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,6 @@ The server is prepared to fetch data directly from the backend before rendering
 | [PostCSS](https://github.com/postcss/postcss) | Transform styles with JS plugins. Used to autoprefix CSS |
 | [Stylelint](https://github.com/stylelint/stylelint) | Modern CSS linter that helps you enforce consistent conventions and avoid errors in your stylesheets. |
 | [Redux Persist](https://github.com/rt2zz/redux-persist) | Persist and rehydrate your redux store |
-| [Isomorphic Fetch](https://github.com/matthew-andrews/isomorphic-fetch) |  Is a Promise-based mechanism for programatically making web requests in the browser. |
```
Comment (mato2593) [inline README.md]: We should add Axios here 😬

File: src/utils/parseError.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * 3- `{ errors: { base: ["User not found"]}}` returns `"User not found"`
+ *
+ * 4- `{ errors: { firstName: ["can't be blank"]}}` returns `"firstName can't be blank"`
+ *
+ * 5- `{ errors: ['Some error'] }` returns `"Some error"`
+ */
+export default data => {
+  if (!data) return 'Something went wrong';
+
+  const { error, errors } = data;
```
Comment (agustito37) [inline src/utils/parseError.js]:
```suggestion
  const { error, errors, message } = data;
```

File: src/utils/parseError.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  if (error) return error;
```
Comment (agustito37) [inline src/utils/parseError.js]: -

File: src/utils/parseError.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+  return parseErrors(errors);
```
Comment (agustito37) [inline src/utils/parseError.js]:
```suggestion
  return error || message || parseErrors(errors);
```

Comment (agustito37) [reply src/utils/parseError.js]: Not necessary thought :)

**Pull request merged**